### ISSUE TITLE
Regression: wizard ids changed in lockhart app

### ISSUE THREAD

**Issue by ren-zhijun-oracle** [eclipse-ee4j/mojarra#1007]
build: b37

There are 9 failures in lockhart app, wizard related tests. These tests were
passing on b35 but are failing on b37.

The following item ids have changed between these builds:

b35:
<span id="form1:wizard1:step1:sv_accttype:email_span" class="RbSpn">
<input type="radio" id="form1:wizard1:step1:sv_accttype:email" name="account"
value="EmailAccount" />

b37:
<span id="form1:wizard1:step1:form1:wizard1:step1:sv_accttype:email_span"
class="RbSpn">
<input type="radio"
id="form1:wizard1:step1:form1:wizard1:step1:sv_accttype:email" name="account"
value="EmailAccount" />

Basically "form1:wizard1:step1" are repeated in b37\. The app can be found at
/net/goverdhan/export/s1as-sqe/tmp/lockhart-testapp.war.
#### Environment
Operating System: All
Platform: All
#### Affected Versions
[current]
